discovery_engine_core/web-api/src/lib.rs

```diff
@@ -12,25 +12,20 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+mod db;
 mod elastic;
-mod handlers;
+mod embedding;
+mod error;
+mod ingestion;
+mod load_config;
+mod logging;
+mod middleware;
 mod models;
-mod routes;
-mod state;
-mod storage;
+mod personalization;
+mod server;
+mod utils;
 
-pub use crate::{
-    elastic::{Config as ElasticConfig, ElasticDocumentData, ElasticState, GenericResponse},
-    models::{
-        DocumentId,
-        DocumentProperties,
-        DocumentProperty,
-        DocumentPropertyId,
-        Error,
-        UserId,
-        COUNT_PARAM_RANGE,
-    },
-    routes::{api_routes, get_health},
-    state::{AppState, InitConfig},
-    storage::UserState,
-};
+pub use error::application::{ApplicationError, Error};
+pub use ingestion::Ingestion;
+pub use personalization::Personalization;
+pub use server::run;
```

Comment:
:art: with a `crate::{` prefix the imports would be grouped nicely.

Reply:
I will add this to 

- #691